htdocs/core/class/html.formsetup.class.php

```diff
@@ -835,7 +835,11 @@ public function generateInputField()
 		} elseif ($this->type== 'color') {
 			$out.=  $this->generateInputFieldColor();
 		} elseif ($this->type == 'yesno') {
-			$out.= $this->form->selectyesno($this->confKey, $this->fieldValue, 1);
+			if (!empty($conf->use_javascript_ajax)) {
```

Comment:
Do you have an example of form popup and what is result (can you provide a screenshot) ?

Reply:
@eldy it's not in popups it's in modules setup pages

Reply:
Sorry, can you provide an example of screen (screenshot) where we can test this ?

Reply:
Befor
![image](https://user-images.githubusercontent.com/34568357/185879454-74e1aaa9-9b30-4387-8fa5-4f30cfde8134.png)

After
![image](https://user-images.githubusercontent.com/34568357/185879264-bf7b04bb-b7e7-4820-9e52-82f767e5d52c.png)